grpcproxy/grpcproxy_test.go

```diff
@@ -0,0 +1,53 @@
+// +build !race
+
+package grpcproxy
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	gr "runtime"
+	"testing"
+
+	"strings"
+
+	"github.com/grpc-ecosystem/grpc-gateway/runtime"
+	"github.com/satori/go.uuid"
+)
+
+func generatePlugin() (string, error) {
+	pName := fmt.Sprintf("%v.so", uuid.NewV4().String())
+	fileArg := fmt.Sprintf("-o=%v", pName)
+
+	cmd := exec.Command("go", "build", "--tags='dummy'", "--buildmode=plugin", fileArg)
```

Comment:
This isn't run in a shell, so remove the quotes around `dummy` (otherwise you really are using `'dummy'` with quotes as the build tag)